addons/stock/static/src/widgets/popover_widget.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+    <div t-name="stock.leadDaysPopOver" owl="1">
+        <h2>Lead Times</h2>
+        <hr/>
+        <p>
+            The forecasted stock on the <b><t t-esc="props.lead_days_date"/></b>
+            is <t t-if="props.qty_to_order &lt;= 0"><b><t t-esc="props.qty_forecast"/> <t t-esc="props.product_uom_name"/></b></t><t t-else="">
+            below the inventory <b>minimum of <t t-esc="props.product_min_qty"/> <t t-esc="props.product_uom_name"/>
+            : <t t-esc="props.qty_to_order"/> <t t-esc="props.product_uom_name"/> should be replenished</b> to reach the maximum of
+            <t t-esc="props.product_max_qty"/> <t t-esc="props.product_uom_name"/>.</t>
```

Comment:
For all the qty value in props, instead of getting them from the props to display them, it should be more readable to have a getter in the component code.
For example, instead of `<t t-esc="props.product_max_qty"/> <t t-esc="props.product_uom_name"/>`, in the template:
```xml
<t t-esc="productMaxQty"/>
```
and in the JS code:
```javascript
get productMaxQty() {
    return this.displayUOM ?
        `${this.props.product_max_qty} ${this.props.product_uom_name}` :
        this.props.product_max_qty;
}
```

And to know if we should display UoM or not, in the setup:
```javascript
const user = useService('user');
onWillStart(async () => {
    this.displayUOM = await user.hasGroup('uom.group_uom');
});
```